lib/data/replacements.json

```
@@ -211,6 +212,18 @@
       }
     ]
   },
+  "eslint-plugin-eslint-comments-to-maintained-fork": {
+    "description": "Replace `eslint-plugin-eslint-comments` with a maintained fork: `@eslint-community/eslint-plugin-eslint-comments`.",
+    "packageRules": [
+      {
+        "matchDatasources": ["npm"],
+        "matchPackageNames": ["eslint-plugin-eslint-comments"],
+        "matchCurrentVersion": "^4.0.0",
```

```suggestion
        "matchCurrentVersion": "^3.2.0",
```

Last version on https://www.npmjs.com/package/eslint-plugin-eslint-comments